src/views/partials/home/card.hbs

```diff
@@ -3,7 +3,7 @@
     class="home-card card-body d-flex flex-column align-items-center justify-content-end text-decoration-none"
     style="background-image: url({{image}}); background-position: center; background-size: cover">
     <p class="h3 d-inline-block p-2 text-dark bg-light">{{title}}</p>
-    <p class="h6 d-inline-block p-2 text-dark bg-light">{{{text}}}</p>
+    <p class="h6 d-inline-block p-2 text-dark bg-light">{{text}}</p>
```

Comment:
There was some formatted text, are you sure you want to escape formatting?

Reply:
I thought `Author | Date` were supposed to be plain text :)